src/menu.js

```diff
@@ -0,0 +1,81 @@
+const menu = () => {
```

Comment:
This is a cool shortcut to get a project, but does not really test your knowledge of JavaScript and Using JS modules. You are actually required to have tabbed content (Main, Menu and Contact (you are missing contact) )where each content is stored in its own module and exported. So Main would be its own module, menu and contact the same.

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)